### PR TITLE
Helics cosimulation: running different helics scripts on AWS

### PR DESCRIPTION
**Pull request by nikolayilyin**
A way to run any configuration of helics scripts with custom parameters without changing lambda code

- - -
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/3695)

### CODE REVIEW

**Comment by nikolayilyin** [comment]
test!

File: gradle.deploy.properties [hunk 1]
```diff
@@ -1,49 +1,37 @@
...
-#r5.12xlarge (48/384)   -> 3 -> $3.024 per Hour
-#c5.18xlarge (72/144)   -> 5 -> $3.06 per Hour
-#c5d.18xlarge (72/144)  -> 5 -> $3.456 per Hour
-#r5d.12xlarge (48/384)  -> 3 -> $3.456 per Hour
-#h1.16xlarge (64/256)   -> 1 -> $3.744 per Hour
-
-#m5.24xlarge (96/384)   -> 2 -> $4.608 per Hour
-#i3.16xlarge (64/488)   -> 1 -> $4.992 per Hour
-#i3.metal (72/512)      -> 5 -> $4.992 per Hour
+# budgetOverride=false
```
Comment (dimaopen) [inline gradle.deploy.properties:27]: Removing prices is intentional?

File: aws/src/main/python/beam_lambda/lambda_function.py [hunk 2]
```diff
@@ -56,12 +36,22 @@
...
+  -       sudo cp /var/log/cloud-init-output.log "$finalPath"
+  -       sudo cp /home/ubuntu/git/beam/thread_dump_from_RunBeam.txt.gz "$finalPath"    
+
+  -       sudo gzip /home/ubuntu/cpu_ram_usage.csv
+  -       sudo cp /home/ubuntu/cpu_ram_usage* "$finalPath"
+
+  -       geminiFinalPath="${finalPath}/gemini_output"
+  -       mkdir "$geminiFinalPath"
+  -       sudo cp /home/ubuntu/git/beam/src/main/python/gemini/cosimulation/*.log "$geminiFinalPath"
+  -       sudo cp /home/ubuntu/git/beam/src/main/python/gemini/cosimulation/*.txt "$geminiFinalPath"
```
Comment (dimaopen) [inline aws/src/main/python/beam_lambda/lambda_function.py]: Maybe somehow find these files. Bash can find them if they exist. In this case we could awoid errors from gzip when no files are presented.

File: gradle.deploy.properties (hunk 1 above)
Comment (nikolayilyin) [reply gradle.deploy.properties:27]: yes, the prices might change in the near future and we usually use very few types of instances anyway

**Comment by nikolayilyin** [comment]
test!

**Review by dimaopen — APPROVED** [review]

**Comment by nikolayilyin** [comment]
test!

**Pull request merged**